aws/container-linux/kubernetes/security.tf

```diff
@@ -199,11 +209,21 @@ resource "aws_security_group" "worker" {
 resource "aws_security_group_rule" "worker-ssh" {
   security_group_id = "${aws_security_group.worker.id}"
 
-  type        = "ingress"
-  protocol    = "tcp"
-  from_port   = 22
-  to_port     = 22
-  cidr_blocks = ["0.0.0.0/0"]
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 22
+  to_port                  = 22
+  source_security_group_id = "${aws_security_group.controller.id}"
```

Comment:
Don't think this is necessary, controllers won't initiate SSH connections

Reply:
Removed.